styles/trinket/catppuccin.user.css

```
@@ -0,0 +1,1455 @@
+
+      .ace_cursor {
+        color: @text;
+      }
+      .ace_marker-layer .ace_selection {
+        background: fade(@accent-color, 30%);
+      }
+      .ace_fold {
+        background-color: @surface0;
+        border-color: @surface1;
+      }
+
+      .ace_constant.ace_language,
+      .ace_keyword,
+      .ace_meta,
+      .ace_variable.ace_language {
+        color: @mauve;
+      }
+      .ace_line .ace_identifier:not(:first-of-type) {
+        color: @blue;
+      }
+      .ace_paren {
+        color: @red;
+      }
+      .ace_constant.ace_numeric {
+        color: @peach;
+      }
+      .ace_entity.ace_other.ace_attribute-name,
+      .ace_support.ace_constant,
+      .ace_support.ace_function {
+        color: @teal;
+      }
+      .ace_entity.ace_name.ace_tag,
+      .ace_support.ace_class,
+      .ace_support.ace_type {
+        color: @blue;
+      }
+      .ace_storage {
+        color: @peach;
+      }
+      .ace_string {
+        color: @green;
+      }
+      .ace_comment {
+        color: @surface2;
+      }
+
+      .ace_indent-guide {
+        @svg: escape(
+          '<svg width="1" height="2" xmlns="http://www.w3.org/2000/svg"><path stroke="@{surface0}" d="M0 0v2"/></svg>'
+        );
+
+        background-image: url("data:image/svg+xml;utf8,@{svg}");
+      }
+    }
+
+    #ace_settingsmenu {
+      background-color: @mantle !important;
+      box-shadow: none !important;
+      color: @subtext0;
+    }
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    body {
+      background-color: @base !important;
+      color: @text !important;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      color: @text !important;
+    }
+
+    a {
+      color: @accent-color;
+      text-decoration-color: @accent-color;
+
+      &:hover {
+        color: @accent-color;
+      }
+    }
+
+    hr,
+    fieldset {
+      border-color: @surface0 !important;
+    }
+
+    footer {
+      border-top-color: @surface0 !important;
+      background-color: @mantle !important;
+    }
+
+    code {
+      background-color: @mantle !important;
+      color: @text !important;
+    }
+
+    select {
+      background-color: @crust !important;
+      border-color: @surface0 !important;
+      color: @text !important;
+    }
+
+    iframe[frameborder] {
+      border-color: @surface0 !important;
+    }
+
+    label {
+      color: @subtext0 !important;
+    }
+
+    table {
+      background-color: @base !important;
+      border-color: @surface0 !important;
+
+      th,
+      td {
+        color: @text !important;
+      }
+
+      tr.even,
+      tr.alt,
+      tr:nth-of-type(2n) {
+        background-color: @mantle !important;
+      }
+
+      thead {
+        background-color: @mantle !important;
+      }
+    }
+
+    input:not([type]),
+    input[type="text"],
+    input[type="password"],
+    input[type="email"],
+    textarea {
```

```suggestion
    input,
    textarea {
```
I believe this would be the same no?

---

yeah i'm pretty sure there's never any color input or whatever